feature-utils/poly-look/src/visualisations/charts/chart-utils.js

```diff
@@ -0,0 +1,6 @@
+export function mapToChartDataArray(map) {
```

Comment:
Would it be possible to add a test for this new function? All `poly-look` is well covered (ish) and I'd like to keep it that way...